### ISSUE TITLE
Scoverage-maven no longer works with Scala-maven-plugin version 4+

### ISSUE THREAD

**Issue by aseigneurin** [scoverage/scoverage-maven-plugin#68]
The following error appears:

```
Execution scala-compile-first of goal net.alchim31.maven:scala-maven-plugin:4.0.1:compile failed: A required class was missing while executing net.alchim31.maven:scala-maven-plugin:4.0.1:compile: scala/xml/NamespaceBinding
```

This is when using:
- `scoverage-maven-plugin` version `1.3.0`
- `scala-maven-plugin` version `4.0.1`

It appears to me that `scala-maven-plugin` no longer depends on `scala-xml` (I read something along these lines in https://github.com/scoverage/scalac-scoverage-plugin/issues/225.

Let me know if I can provide additional information to help fix this issue.

**Comment by gslowikowski** [comment]
Hi @aseigneurin 

Quick workaround is to use latest (`1.4.0-M5` now) [scalac plugin version](http://scoverage.github.io/scoverage-maven-plugin/1.3.0/pre-compile-mojo.html#scalacPluginVersion), e.g.:
```
            <plugin>
                <groupId>org.scoverage</groupId>
                <artifactId>scoverage-maven-plugin</artifactId>
                <version>${scoverage.plugin.version}</version>
                <configuration>
                    <scalacPluginVersion>1.4.0-M5</scalacPluginVersion>
                    <highlighting>true</highlighting>
                </configuration>
            </plugin>
```

Proper solution would be to release Maven plugin version `1.4.0-M5` (using this version of scalac plugin internally).

**Comment by gslowikowski** [comment]
Sorry, unfortunately the workaround does not work. Report is not generated.

**Comment by gslowikowski** [comment]
`1.4.0-SNAPSHOT` snapshot deployed. Please verify if it works for you with new `scala-maven-plugin` versions.

**Comment by gslowikowski** [comment]
Version `1.4.0-M5` released. M5, because it supports Scala `2.13.0-M5`.

**Comment by aseigneurin** [comment]
Sorry for the delay. I can confirm the issue is resolved with `1.4.0-M5`.

Thanks for your help.
